From acab9fbb901b91364a0e4e697e6806e2c438954e Mon Sep 17 00:00:00 2001
From: Jiang Lu <lu.jiang@windriver.com>
Date: Mon, 19 Aug 2013 16:28:00 +0800
Subject: [PATCH] powerpc/dpaa: fix mdio dependency

commit e6d758f1e33aa1fe63288636086484239c53731b
(dpaa_eth: use FSL_DPAA_ETH prefix for all config options)
use FSL_DPAA_ETH prefix,
so the mdio dependency should also be changed accordingly.

Signed-off-by: Shaohui Xie <Shaohui.Xie@freescale.com>
Change-Id: I97697d33dbd1f8da2b504bb45b8d45aa489a0a96
Reviewed-on: http://git.am.freescale.net:8181/2081
Reviewed-by: Bucur Madalin-Cristian-B32716 <madalin.bucur@freescale.com>
Reviewed-by: Hamciuc Bogdan-BHAMCIU1 <bogdan.hamciuc@freescale.com>
Reviewed-by: Zang Tiefei-R61911 <tie-fei.zang@freescale.com>
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>

[origin patch is from QorIQ-SDK-V1.4-20130625-yocto.]
Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 arch/powerpc/platforms/85xx/Kconfig |   22 +++++++++++-----------
 1 files changed, 11 insertions(+), 11 deletions(-)

diff --git a/arch/powerpc/platforms/85xx/Kconfig b/arch/powerpc/platforms/85xx/Kconfig
index 0107ec8..d1c6e74 100644
--- a/arch/powerpc/platforms/85xx/Kconfig
+++ b/arch/powerpc/platforms/85xx/Kconfig
@@ -255,8 +255,8 @@ config P3041_DS
 	select HAS_FSL_PAMU
 	select HAS_FSL_QBMAN
 	select HAS_FSL_PME
-	select MDIO_BUS_MUX if DPA_ETH
-	select MDIO_BUS_MUX_MMIOREG if DPA_ETH
+	select MDIO_BUS_MUX if FSL_DPAA_ETH
+	select MDIO_BUS_MUX_MMIOREG if FSL_DPAA_ETH
 	help
 	  This option enables support for the P3041 DS board
 
@@ -286,7 +286,7 @@ config P4080_DS
 	select HAS_FSL_PAMU
 	select HAS_FSL_QBMAN
 	select HAS_FSL_PME
-	select MDIO_BUS_MUX if DPA_ETH
+	select MDIO_BUS_MUX if FSL_DPAA_ETH
 	select MDIO_BUS_MUX_GPIO
 	help
 	  This option enables support for the P4080 DS board
@@ -306,8 +306,8 @@ config B4_QDS
 	select HAS_RAPIDIO
 	select PPC_EPAPR_HV_PIC
 	select HAS_FSL_PAMU
-	select MDIO_BUS_MUX if DPA_ETH
-	select MDIO_BUS_MUX_MMIOREG if DPA_ETH
+	select MDIO_BUS_MUX if FSL_DPAA_ETH
+	select MDIO_BUS_MUX_MMIOREG if FSL_DPAA_ETH
 	help
 	  This option enables support for the B4 QDS board
 
@@ -327,8 +327,8 @@ config P5020_DS
 	select HAS_FSL_PAMU
 	select HAS_FSL_QBMAN
 	select HAS_FSL_PME
-	select MDIO_BUS_MUX if DPA_ETH
-	select MDIO_BUS_MUX_MMIOREG if DPA_ETH
+	select MDIO_BUS_MUX if FSL_DPAA_ETH
+	select MDIO_BUS_MUX_MMIOREG if FSL_DPAA_ETH
 	help
 	  This option enables support for the P5020 DS board
 
@@ -349,8 +349,8 @@ config P5040_DS
 	select HAS_FSL_PAMU
 	select HAS_FSL_QBMAN
 	select HAS_FSL_PME
-	select MDIO_BUS_MUX if DPA_ETH
-	select MDIO_BUS_MUX_MMIOREG if DPA_ETH
+	select MDIO_BUS_MUX if FSL_DPAA_ETH
+	select MDIO_BUS_MUX_MMIOREG if FSL_DPAA_ETH
 	help
 	  This option enables support for the Freescale P5040 DS
 	  ("Super Hydra") reference board.
@@ -369,8 +369,8 @@ config T4240_QDS
 	select HAS_FSL_PAMU
 	select HAS_FSL_QBMAN
 	select HAS_FSL_PME
-	select MDIO_BUS_MUX if DPA_ETH
-	select MDIO_BUS_MUX_MMIOREG if DPA_ETH
+	select MDIO_BUS_MUX if FSL_DPAA_ETH
+	select MDIO_BUS_MUX_MMIOREG if FSL_DPAA_ETH
 	help
 	  This option enables support for the T4240 QDS board
 
-- 
1.7.0

